x11-misc/gpaste/gpaste-3.36.3.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION="0.30"
+VALA_USE_DEPEND="vapigen"
+
+inherit meson vala gnome2-utils
+
+DESCRIPTION="Clipboard management system"
+HOMEPAGE="https://github.com/Keruspe/GPaste"
+SRC_URI="https://github.com/Keruspe/GPaste/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+introspection +gnome vala systemd"
+REQUIRED_USE="
+	vala? ( introspection )
+	gnome? ( introspection )
+"
+
+DEPEND="
+	dev-libs/appstream-glib
+	>=dev-libs/glib-2.48:2
+	introspection? (
+		>=dev-libs/gobject-introspection-1.48.0
+	)
+	sys-apps/dbus
+	>=x11-libs/gdk-pixbuf-2.34:2
+	>=x11-libs/gtk+-3.20:3
+	x11-libs/libX11
+	x11-libs/libXi
+	gnome? (
+		media-libs/clutter
+		x11-libs/pango
+	)
```

Comment:
This fails with `-gnome`

I'm basically trying to merge this on an empty stage3 image with first `-gnome -vala -introspection -systemd`. Since I don't have any of those enabled on my desktop either.

And when that works, I'd try with every USE flag enabled except systemd. Just FYI how you should test this :) 

```
Run-time dependency mutter-clutter-6 found: NO (tried pkgconfig and cmake)

meson.build:40:2: ERROR: Dependency "mutter-clutter-6" not found, tried pkgconfig and cmake

A full log can be found at /var/tmp/portage/x11-misc/gpaste-3.36.3/work/gpaste-3.36.3-build/meson-logs/meson-log.txt
 * ERROR: x11-misc/gpaste-3.36.3::testworld failed (configure phase):
 *   (no error message)
```

Reply:
Perhaps the old gpaste ebuild was wrong, or perhaps gpaste upstream moved to using mutter over clutter libs at some point, but replacing clutter dep with mutter has resolved this issue for me in my testing. However, I was only able to reproduce this issue when enabling the introspection use flag.